proposals/exception-handling/Exceptions.md

```diff
@@ -311,6 +310,39 @@ catching instructions (`catch`/`catch_all`/`delegate`) come after the
 same as if the `try` has catches but none of the catches are able to handle the
 exception.
 
+If `delegate`'s immediate argument is the depth of block-like structures
+(blocks, loops, and trys) + 1, i.e., the function-level block, it is considered
+to be delegating the exception to the caller of the current function. For
+example:
+```
+try $l1
+  try
+    call $foo
+  delegate 1  ;; delegate to the caller
+catch
+  ...
+catch_all
+  ...
+end
```

Comment:
To make the example complete I think it should be wrapped in a function, otherwise it is ambiguous what the outer scope is.

Reply:
Done